### DISCUSSION TITLE
Display History in Multiple Lines Upwards

### DISCUSSION THREAD

**Discussion by winter60** [mulaRahul/keyviz#34] [Ideas]
现在只显示的一行，若配置成多行的话可以同时看到连续的操作键盘记录

**Comment by mulaRahul** [comment]
![append-history](https://user-images.githubusercontent.com/96373135/185865041-14bd8dad-16a0-41db-bdd2-871c79f75e97.png)

The existing "**Append**" feature will be modified to something like the above (_to show the history of pressed keys in multiple lines in an upwards direction_).

**Comment by mulaRahul** [comment]
Added with [v2.0.0a](https://github.com/mulaRahul/keyviz/releases/tag/v2.0.0a).
